pykep/_version.py

```diff
@@ -1 +1,8 @@
+## Copyright 2023, 2024 Dario Izzo (dario.izzo@gmail.com), Francesco Biscani
```

Comment:
Is this file included by mistake?

Reply:
Yes :) will remove before merging well spotted!